HardwareDefinitions/avnet_mt3620_sk_rev2/sample_appliance.json

```diff
@@ -0,0 +1,44 @@
+            "// 'sample hardware' abstraction used by the samples at https://github.com/Azure/azure-sphere-samples.",
+            "// Some peripherals are on-board on the Avnet MT3620 SK, while other peripherals must be attached externally if needed.",
+            "// See https://aka.ms/AzureSphereHardwareDefinitions for more information on how to use hardware abstractions,",
+            "// to enable apps to work across multiple hardware variants."
+        ]
+    },
+
+    "Imports" : [ {"Path": "avnet_mt3620_sk_rev2.json"} ],
+    "Peripherals": [
+        {"Name": "SAMPLE_BUTTON_1", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_USER_BUTTON_A", "Comment": "MT3620 SK: User Button A."},
+        {"Name": "SAMPLE_BUTTON_2", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_USER_BUTTON_B", "Comment": "MT3620 SK: User Button B."},
+        {"Name": "SAMPLE_LED_PWM_CONTROLLER", "Type": "Pwm", "Mapping": "AVNET_MT3620_SK_PWM_CONTROLLER2", "Comment": "MT3620 SK: PWM LED controller"},
+        {"Name": "SAMPLE_LED_PWM_CHANNEL", "Type": "int", "Mapping": "MT3620_PWM_CHANNEL1", "Comment": "MT3620 SK: Channel 1 for the PWM LED1 green. In the app manifest, it is only necessary to request the capability for the PWM Controller, SAMPLE_LED_PWM_CONTROLLER."},
+        {"Name": "SAMPLE_LED", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_USER_LED_RED", "Comment": "MT3620 SK: User LED."},
+        {"Name": "SAMPLE_POTENTIOMETER_ADC_CONTROLLER", "Type": "Adc", "Mapping": "AVNET_MT3620_SK_ADC_CONTROLLER0", "Comment": "MT3620 SK: ADC Potentiometer controller"},
+        {"Name": "SAMPLE_POTENTIOMETER_ADC_CHANNEL", "Type": "int", "Mapping": "MT3620_ADC_CHANNEL1", "Comment": "MT3620 SK: Connect external potentiometer to ADC controller 0, channel 1 using CLICK1 AN. In the app manifest, it is only necessary to request the capability for the ADC Group Controller, SAMPLE_POTENTIOMETER_ADC_CONTROLLER."},
+        {"Name": "SAMPLE_RGBLED_RED", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_USER_LED_RED", "Comment": "MT3620 SK: User LED RED Channel."},
+        {"Name": "SAMPLE_RGBLED_GREEN", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_USER_LED_GREEN", "Comment": "MT3620 SK: User LED GREEN Channel."},
+        {"Name": "SAMPLE_RGBLED_BLUE", "Type": "Gpio", "Mapping": "AVNET_MT3620_SK_USER_LED_BLUE", "Comment": "MT3620 SK: User LED BLUE Channel."},
+        {"Name": "SAMPLE_UART_LOOPBACK", "Type": "Uart", "Mapping": "AVNET_MT3620_SK_ISU0_UART", "Comment": "MT3620 SK: Connect CLICK1 RX (RX) to CLICK1 TX (TX)."},
+        {"Name": "SAMPLE_LSM6DS3_I2C", "Type": "I2cMaster", "Mapping": "AVNET_MT3620_SK_ISU2_I2C", "Comment": "MT3620 SK: Connect external LSM6DS3 to I2C using CLICK1, pin MISO (SDA) and pin MOSI (SCL)."},
+        {"Name": "SAMPLE_LSM6DS3_SPI", "Type": "SpiMaster", "Mapping": "AVNET_MT3620_SK_ISU1_SPI", "Comment": "MT3620 SK: Connect external LSM6DS3 to SPI using CLICK1, pin MISO (MISO), pin SCK (SCLK), pin MOSI (MOSI) and CLICK2 pin CS (CSB)."},
```

Comment:
The AVNET_MT3620_SK_ISU1_SPI isn't defined in the the avnet_mt3620_sk_rev2.json file. Should the SAMPLE_LSM6DS3_SPI be mapped to AVNET_MT3620_SK_ISU1_SPI_CONFIGURABLE_CLICK2 instead and the comment changed accordingly?

Reply:
Good catch.  SAMPLE_LSM6DS3_SPI should be mapped to AVNET_MT3620_SK_ISU0_SPI, not ISU1.  I'll update the sample_appliance.* files and update the pull request.

Reply:
My branch has been updated with this change.  Please let me know if there are any other questions.